code/game/objects/items/twohanded.dm

```diff
@@ -806,3 +806,38 @@
 
 /obj/item/twohanded/bonespear/update_icon()
 	icon_state = "bone_spear[wielded]"
+
+/obj/item/twohanded/binoculars
+	name = "binoculars"
+	desc = "Used for long-distance surveillance."
+	item_state = "binoculars"
+	icon_state = "binoculars"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	var/datum/component/mobhook
+	var/view_range_increase = 7
+
+/obj/item/twohanded/binoculars/wield(mob/user)
+	..()
+	if(wielded)
+		mobhook = user.AddComponent(/datum/component/redirect, list(COMSIG_MOVABLE_MOVED = CALLBACK(src, .proc/unwield, user)))
+		user.visible_message("\The [user] holds \the [src] up to \his eyes.","You hold \the [src] up to your eyes.")
```

Comment:
`"The Citrus Gender holds the"`

wat

lose all the `\the`'s

also pronoun helpers. `user.p_their()`